b9lab-content/2-what-is-cosmos/02-blockchain-and-cosmos.md

```diff
@@ -0,0 +1,154 @@
+
+Let's now look at the problems Cosmos solves.
+
+### How Does Cosmos Solve the Scalability Issue?
+
+As mentioned previously, scalability is a big issue area when it comes to blockchain technology. Cosmos allows applications to scale to millions of users.
+
+Cosmos addresses **two types of scalability**:
+
+* **Horizontal scalability**: Scaling by adding similar machines to the network. When scaling out, horizontally, the network can accept more nodes to participate in the state replication, consensus observation, and any activity that queries the state.
+* **Vertical scalability**: Scaling by improving the network's components so as to increase its computational power. When scaling up, vertically, the network can accept more transactions, and any activity that modifies the state.
+
+In a blockchain context, vertical scalability is typically achieved through the optimization of the consensus mechanism and applications running on the chain. On the consensus side, Cosmos achieves vertical scalability with the help of the Tendermint BFT. For instance, the Cosmos Hub currently conducts transactions in seven seconds. In a single blockchain context, the only remaining bottleneck is then the application.
+
+Consensus mechanism and application optimization of one's blockchain can only take one so far. To overcome the limits of vertical scalability, Cosmos' multi-chain architecture allows for **one application to run in parallel** on different chains, all the while being operated by the same validator set. This inter-chain horizontal scalability theoretically allows for infinite vertical-like scalability.
+
+<HighlightBox type="info">
+
+A **validator** is one or more cooperating computers that participate in the consensus by, among other things, creating blocks.
+
+</HighlightBox>
+
+### How Does Cosmos Promote Sovereignty?
+
+Applications deployed on general-purpose blockchains all share the same underlying environment. When a change in the application needs to be made, it not only depends on the governance structures of the application but also on that of the environment, i.e. the governance mechanisms set by the protocol on which the application builds. Thus, the chain's governance limits the application's sovereignty. For this reason, it is often called a **two-layer governance**.
+
+Cosmos resolves this issue as one can build a blockchain tailored to the application. There are no limits to the application's governance if every chain is maintained by its own set of validators. Cosmos follows a design of a **one-layer governance**.
+
+### How Does Cosmos Improve User Experience?
+
+In the world of traditional general-purpose blockchains, application design and efficiency are limited for blockchain developers. They have to find the simplistic use case the blockchain is designed for.
```

Comment:
It's not really clear that Ethereum's use case is "simplistic". Developers can still build basically anything. It's just more expensive and clunky and not tailored to your needs and so on